devel/libcoverart/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        metabrainz libcoverart 1.0.0 release-
+github.tarball_from releases
+categories          devel
+platforms           darwin
+maintainers         nomaintainer
+license             LGPL-2.1+
+
+description         C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+long_description    libcoverart is a C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+homepage            https://musicbrainz.org/doc/libcoverart
+
+checksums           rmd160  2c3d15b3399b16b842cde85fa02401caf72e4cdb \
+                    sha256  5d9d4a4c46f35754e016affab83957961a364d7a4c7d64447f73bff22a300f44 \
+                    size    57933
+
+depends_lib         \
+                    port:neon \
+                    port:jansson \
+                    port:libxml2
+
+depends_build       \
+                    port:pkgconfig
```

Comment:
You should use `depends_build-append` here so that you don't overwrite dependencies from portgroups.

Reply:
Got it! That would by why I had to put the PortGoup cmake 1.0 below this to get it to build. Fixed now!